src/Compute/Compute.Test/LiveTests/TestLiveScenarios.ps1

```diff
@@ -0,0 +1,33 @@
+Invoke-LiveTestScenario -Name "VM.NewVM Test" -Description "Test create new VM" -ScenarioScript `
+{
+    param ($rg)
+
+    $rgName = $rg.ResourceGroupName
+    $name = New-LiveTestResourceName
+
+    $vm = Get-AzVM -ResourceGroupName $rgName -Name $name 
+    if ($null -eq $vm) {
+        $actual = New-AzVM -Name $name -Credential (Get-Credential)
+    }
+    Assert-AreEqual $name $actual.Name
+
+    Assert-AreEqual $false $actual.EnabledForDeployment
+    Assert-Null $actual.Zone "By default Zone should be null"
+}
+
+Invoke-LiveTestScenario -Name "VM.RemoveVM Test" -Description "Test remove VM" -ResourceGroupLocation "eastus" -ScenarioScript `
+{
+    param ($rg)
+
+    $rgName = $rg.ResourceGroupName
+    $name = New-LiveTestResourceName
+
+    New-AzVM -Name $name -Credential (Get-Credential)
```

Comment:
Get-Credential is an interactive command which is not allowed in the live test. Please use a non-interactive command instead to create the VM.

Reply:
modified